modules/UI/videolayout/VideoLayout.js

```diff
@@ -361,11 +361,15 @@ var VideoLayout = {
     },
 
     isPinned (id) {
+        const pinnedId = this.getPinnedId();
+
         return (pinnedId) ? (id === pinnedId) : false;
     },
 
     getPinnedId () {
-        return pinnedId;
+        const participant = getPinnedParticipant(APP.store.getState());
+
+        return participant ? participant.id : null;
```

Comment:
Get the pinned id from redux instead of within the private scope. This moves the source of truth in one place.